cli-feed-v4.json

```diff
@@ -6632,7 +6632,7 @@
           "netcore3": {
             "displayInfo": {
               "displayName": ".NET Core 3.1",
-              "hidden": false,
+              "hidden": true,
```

Comment:
We can't update an existing release version. We instead need to create a new entry with field hidden.

Reply:
same for other changes

Reply:
Created 3.47.1 from 3.47.0 and set hidden true for .NET Core 3.1. The .NET core 3.1 entry is only present in the c4 feed json.